ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerview.js

```diff
@@ -16,7 +16,7 @@
 (function() {
 
     "use strict";
-    class DatePicker extends FormView.FormFieldBase {
+    class DatePicker extends FormView.FormDatePicker {
```

Comment:
Why did you create a new class here when there is no use-case ?

Reply:
The use case is that I need to use the parse function of af-core which cannot be imported in datepickerview.js

Reply:
can we not export it like we export createFormInstance ?